simulator/channel_sim.cc

```diff
@@ -8,8 +8,12 @@
 
 #include "datatype_conversion.h"
 
-static bool running = true;
+using namespace arma;
```

Comment:
It seems you are trying to remove this namespace in other files, should this also be removed here?

Reply:
You are probably right,  I attempted to remove all the using namespace directives from the header files due to the following rule "Don't put namespace aliases in your public API".  However, looking at the namespace guidelines "Do not use using-directives" it might be more correct to remove them all.

Reply:
Removed all using namespace directives